packages/launchpad/src/migration/MajorVersionWelcome.vue

```diff
@@ -0,0 +1,168 @@
+<template>
+  <div class="bg-no-repeat bg-cover h-screen min-h-700px lp-wrapper">
+    <div
+      ref="wrapper"
+      class="border-transparent rounded mx-auto bg-gray-50/50 border-4px max-w-80vw top-7vh w-716px relative overflow-hidden"
+    >
+      <div
+        ref="scroller"
+        class="bg-white rounded-b max-h-72vh pb-90px overflow-scroll"
+      >
+        <div class="h-full overflow-scroll">
+          <div class="p-16px">
+            <h1 class="font-medium mt-4px text-center mb-32px tracking-tighter text-22px text-gray-1000">
+              {{ t('majorVersionWelcome.title') }}
+            </h1>
+            <div class="mb-16px">
+              <ExternalLink
+                href="https://on.cypress.io/changelog#11-0-0"
+                class="font-bold text-indigo-500"
+              >
+                11.0.0
+              </ExternalLink>
+              <span class="font-light pl-10px text-gray-500 text-14px">
+                Released {{ versionReleaseDates['11'] }}
+              </span>
+            </div>
+            <div class="children:mb-16px">
+              <h2 class="font-bold text-18px text-jade-1000">
+                Component Testing Released
+              </h2>
+
+              <p>
+                Component Testing is now generally available for projects using React, Vue, Angular, and Next.js!
```

Comment:
This might just be me, but it makes makes my brain feel better to group React and Next.js together since they are related.

```suggestion
                Component Testing is now generally available for projects using React, Next.js, Angular, and Vue!
```

Reply:
Updated 👍